Common/Sources/Insights/HyperwalletInsights.swift

```diff
@@ -145,3 +137,63 @@ public class HyperwalletInsights: HyperwalletInsightsProtocol {
         }
     }
 }
+
+/// A helper class to build the `ErrorInfo` instance.
+public class ErrorInfoBuilder {
+    private let type: String
+    private let message: String
+    private var fieldName = ""
+    private var description = Thread.callStackSymbols.joined(separator: "\n")
```

Comment:
This can be let

Reply:
@ddhar-hw why var?

Reply:
The reason I used `var` was because I was not sure if we would always set stack trace in description - if that is the case - sure.

Reply:
Going forward if there's any uncertainty about the requirement then let's discuss that and be sure fo the implementation. This needs to be let and the `public func description(_ description: String) -> ErrorInfoBuilder` method can be removed.